platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/InitScriptApi.kt

```diff
@@ -68,13 +63,6 @@ abstract class InitScriptApi(
     protected
     abstract val fileOperations: FileOperations
 
-    protected
-    abstract val processOperations: ProcessOperations
```

Comment:
❔ @gradle/bt-kotlin-dsl-maintainers this is a breaking change, but it looks like this class, along with its peers is going away in 9.0, right?

Reply:
I think they are fine to remove.